### ISSUE TITLE
RS485 be connected to indoor or outdoor

### ISSUE THREAD

**Issue by guohj917** [Bunicutz/ESP32_Midea_RS485#14]
Should RS485 be connected to the PQ terminal of the central air conditioning indoor unit or the XY terminal of the external unit

**Comment by Flachzange** [comment]
What is a PQ terminal?

It needs to go to the XY-Bus. I only have bus access at my indoor units but if you have it available at the outdoor unit, this is to be preferred because you could control multiple indoor units (untested).

**Comment by guohj917** [comment]
我的意思是这个esp32接在室内机还是室外机上，您是接在室内机上实现的吗？有多个室内机就需要接多个esp32？

**Comment by Flachzange** [comment]
Englisch please

**Comment by guohj917** [comment]
Is ESP32 connected to an indoor unit or an outdoor unit? Did you implement it by connecting it to an indoor unit? Do multiple indoor units require multiple ESP32 connections?

**Comment by guohj917** [comment]
This is the circuit diagram of my indoor unit
![mmexport1719277658907](https://github.com/Bunicutz/ESP32_Midea_RS485/assets/85216453/34776681-d078-43db-a9e1-99bd2f7b2254)

**Comment by Flachzange** [comment]
> Is ESP32 connected to an indoor unit or an outdoor unit? Did you implement it by connecting it to an indoor unit? Do multiple indoor units require multiple ESP32 connections?

Yes, I have one ESP32 per Indoor Unit. But there are some outdoor units which also have access to the XY Bus. In this case it is possible to connect to the outdoor unit.
